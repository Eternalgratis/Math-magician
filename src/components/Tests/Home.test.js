import Renderer from 'react-test-renderer';
import Home from '../Home';

it('Renders Home component', () => {
  const tree = Renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
