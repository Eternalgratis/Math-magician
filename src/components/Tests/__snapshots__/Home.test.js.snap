// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Home component 1`] = `
<section
  className="home-page"
>
  <div
    className="title"
  >
    <h3>
      Welcome to our page!
    </h3>
  </div>
  <div
    className="text"
  >
    <p
      className="p-one"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse at dui a neque maximus ullamcorper id eu nisi. Vivamus id tortor in sapien dignissim finibus. Curabitur faucibus, libero at tempor porta, augue lectus molestie massa, ac viverra mi arcu ac tellus. Etiam non placerat neque. Nullam id nibh maximus, viverra leo sed, tristique ligula. Praesent eleifend mollis vestibulum. Mauris vel lacus a neque congue ultrices. Curabitur in ligula sit amet elit accumsan gravida eget id dui.
    </p>
    <p
      className="p-two"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse at dui a neque maximus ullamcorper id eu nisi. Vivamus id tortor in sapien dignissim finibus. Curabitur faucibus, libero at tempor porta, augue lectus molestie massa, ac viverra mi arcu ac tellus. Etiam non placerat neque. Nullam id nibh maximus, viverra leo sed, tristique ligula. Praesent eleifend mollis vestibulum. Mauris vel lacus a neque congue ultrices. Curabitur in ligula sit amet elit accumsan gravida eget id dui. Proin egestas mi erat, in dictum urna pulvinar ac. Aenean fringilla posuere nibh consectetur tincidunt. Proin at risus quis quam posuere varius sed ut lorem.
    </p>
  </div>
</section>
`;
